src/skmultiflow/trees/split_criterion/intra_cluster_variance_reduction_split_criterion.py

```diff
@@ -37,4 +36,4 @@ def compute_SD(dist):
         sum = dist[1]
         sum_sq = dist[2]
 
-        return np.sum((sum_sq - (sum * sum) / N) / (N - 1))
+        return np.mean((sum_sq - ((sum * sum) / N)) / (N - 1))
```

Comment:
Was this a bug? We need to document the rationale for this change and its impact.

Reply:
Yes, it was. I had forgotten to create a PR for this aspect.

I found this bug while performing some experiments. As much as proportionally the values are similar, this indeed might impact in the ICVR calculation.

For clearness, I will submit a separate minor PR for this aspect. What do you think?

 How do I must proceed to document this aspect? Adding it to the PR description?
Thank you for pointing it out.

Reply:
This has been addressed in its own PR #152.